src/Microsoft.IdentityModel.Tokens/InternalValidators.cs

```diff
@@ -72,7 +72,7 @@ internal static void ValidateLifetimeAndIssuerAfterSignatureNotValidatedJwt(Secu
             }
 
             if (validLifetime && validIssuer)
-                throw LogHelper.LogExceptionMessage(new SecurityTokenSignatureKeyNotFoundException(LogHelper.FormatInvariant(TokenLogMessages.IDX10501, kid, exceptionStrings, securityToken)));
+                throw LogHelper.LogExceptionMessage(new SecurityTokenSignatureKeyNotFoundException(LogHelper.FormatInvariant(TokenLogMessages.IDX10501, kid, LogHelper.MarkAsNonPII(numKeysInTokenValidationParameters), LogHelper.MarkAsNonPII(numKeysInConfiguration), exceptionStrings, securityToken)));
```

Comment:
consider stacking parameters as above.

Reply:
Fixed! https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1786/commits/27f73791bb1e017be2e7bc1a0b4c25cba31b9a78